modules-js/react-fleet/src/form-elements/SelectDropdown.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { SelectHTMLAttributes } from 'react';
+import hash from 'string-hash';
+
+
+interface Props extends SelectHTMLAttributes<HTMLSelectElement> {
```

Comment:
I think if you’re going to use `SelectHTMLAttributes` then you need to pass all unused `props` to the `<select>` element.

If you want to only have some props on `<select>`, then enumerate the explicitly instead of `extends`ing.